### PR TITLE
Replaces cssLiteral (removed) with unsafeCSS 

### PR DESCRIPTION
**Pull request by hyperpress**
Updates style guide by removing reference to cssLiteral (previously removed) for using expressions in static styles with unsafeCSS method.

### CODE REVIEW

**Comment by web-padawan** [comment]
@sorvell @justinfagnani this is important to merge, as the docs are still pointing to `cssLiteral` and the users, who are not aware with LitElement internals, continue to face the issue with incorrect docs.

File: docs/_guide/styles.md [hunk 1]
```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use unsafeCSS to pass non-literal values using only trusted sources:
```
Comment (justinfagnani) [inline docs/_guide/styles.md]: The replacement for `cssLiteral` is just `css`.

File: docs/_guide/styles.md [hunk 2]
```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
...
 
 ```js
-import { LitElement, css, cssLiteral } from 'lit-element';
+import { unsafeCSS } from 'lit-element';
 
-const mainColor = cssLiteral`red`;
+const mainColor = unsafeCSS('red');
```
Comment (justinfagnani) [inline docs/_guide/styles.md]:
```suggestion
const mainColor = css`red`;
```

File: docs/_guide/styles.md (hunk 1 above)
Comment (hyperpress) [reply docs/_guide/styles.md]: Done. Thanks for the clarification.

File: docs/_guide/styles.md [hunk 3]
```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use unsafeCSS to pass non-literal values using only trusted sources:
 
 ```js
-import { LitElement, css, cssLiteral } from 'lit-element';
+import { unsafeCSS } from 'lit-element';
```
Comment (abraham) [inline docs/_guide/styles.md]: `LitElement` is used in the example so it should still be imported.

Comment (hyperpress) [reply docs/_guide/styles.md]: Fixed the import to include LitElement in latest commit.

File: docs/_guide/styles.md [hunk 4]
```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use **css** to pass non-literal values using only trusted sources:
```
Comment (arthurevans) [inline docs/_guide/styles.md]: This appears to be the description for `unsafeCss`. You can't use `css` with non-literal values, and "trusted sources" doesn't make sense for a literal. Correct me if I'm wrong here, @justinfagnani

Comment (hyperpress) [reply docs/_guide/styles.md]: I refactored this a bit to hopefully provide coverage and clarity, but I may be missing something that is planned that will change this, such as #626. Sorry about dragging out what I thought would be a simple doc change. Please advise if I need to address something else here.

**Comment by arthurevans** [comment]
Yeah, I agree it's confusing. 

When you use `css`, you can _only_ use it on a string literal.  This is OK:

```
const mainColor = css`red`;
return css`:host { color: ${mainColor}; }`
```

This is not:

```
const themeColor = 'red';
return = css`:host { color: ${themeColor}; }`;
```

Any expression you interpolate into the `css` string needs to be tagged with either `css` or `unsafeCSS`.  If you want to inject any variable or non-literal into a `css` string, you need to wrap it with `unsafeCSS`.

```
const themeColor = 'red';
return = css`:host { color: ${unsafeCSS(themeColor)}; }`;
```

Or:

```
css`:host { width: ${unsafeCSS(mainWidth + padding)}px; }
```

The outstanding PR adds support for expressions that return a number--in which case you could remove the `unsafeCSS` from the last example.

**Comment by hyperpress** [comment]
@arthurevans Thanks very much for the detailed clarification. If #626 will be merged in time for the next release, I can add a quick example detailing that exemption. If not, I believe the latest commit will suffice as is until that change is made. If you or @justinfagnani can let me know I'll act accordingly. Cheers.

File: docs/_guide/styles.md [hunk 5]
```diff
@@ -111,27 +111,33 @@ To define a static `styles` property:
...
+import { LitElement, unsafeCSS } from 'lit-element';
 
 class MyElement extends LitElement {
   static get styles() {
-    return css`
-    :host {
-      display: ${displayType};
-      color: ${mainColor};
-    }`;
+    return unsafeCSS`
```
Comment (arthurevans) [inline docs/_guide/styles.md]: I don't think you can use unsafeCSS this way. You can only use it as a filter inside an expression.

For example:

```
css`
  :host {
    color: ${unsafeCSS(mainColor)};
  }`;
```

Comment (arthurevans) [reply docs/_guide/styles.md]: i.e., it's not a tag function like `css`, it's a method you can use in expressions.

Comment (katejeffreys) [reply docs/_guide/styles.md]: yep sorry I merged too soon. fixed it

**Comment by katejeffreys** [comment]
@hyperpress @arthurevans @web-padawan 

Sorry for the delay on this & heavy edits to John's PR. If any of y'all get the chance to look over it, that'd be great

**Comment by katejeffreys** [comment]
@justinfagnani @sorvell fyi - thought i should dismiss your reviews because of significant changes to this section. If you get the chance to take another quick look in case I've accidentally reverted a change you asked for that would be super awesome. Sorry for the delay on this.

**Comment by arthurevans** [comment]
LGTM.

**Comment by hyperpress** [comment]
Good to go 👍

**Comment by hyperpress** [comment]
Are w

**Comment by arthurevans** [comment]
Yes, we should. Can you merge this and redeploy, @katejeffreys?

**Comment by e111077** [comment]
ping on status on this pr. #637 is addressed in the contents of this PR

**Pull request merged**